### PR TITLE
759/fix achievement page UI changes

### LINKED ISSUE #759: change achievement page UI according to design
**Describe the bug**

fix achievement page UI according to design

### PR DESCRIPTION
**Pull request by kunwar-11**
# Pull Request Template

## Description

fixed achievement page UI and achievement filtering

Fixes #759 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pages/u/achievement.js [hunk 1]
```diff
@@ -1,134 +1,156 @@
...
+  const [achievement , setAchievement] = useState(achievements)
+
+  const router = useRouter()
+
+  useEffect(() => {
+    console.log(router.query.type)
+      setAchievement(achievements)
+      if(router.query.type){
+        const filtered = achievements.filter(each => each.type.toLowerCase() === router.query.type.toLowerCase())
+        console.log(filtered , "f")
```
Comment (jasirtp) [inline pages/u/achievement.js:129]: remove console

File: pages/u/achievement.js [hunk 2]
```diff
@@ -1,134 +1,156 @@
...
+        console.log(achievement , "a")
```
Comment (jasirtp) [inline pages/u/achievement.js:130]: remove

File: pages/u/achievement.js [hunk 3]
```diff
@@ -1,134 +1,156 @@
...
-]
+];
 
 const Achievement = () => {
+  const [achievement , setAchievement] = useState(achievements)
+
+  const router = useRouter()
+
+  useEffect(() => {
+    console.log(router.query.type)
```
Comment (jasirtp) [inline pages/u/achievement.js:125]: remove console

File: component/layout/achievement/AchievementCategory.jsx [hunk 4]
```diff
@@ -1,28 +1,41 @@
-import React from 'react'
+import {useRouter} from 'next/router'
 
 const AchievementCategory = () => {
+
+  const router = useRouter()
+
+  const achievementFilter = (e) => {
+    console.log(e.target)
```
Comment (jasirtp) [inline component/layout/achievement/AchievementCategory.jsx:8]: remove console

**Comment by kunwar-11** [comment]
all console logs removed

**Pull request merged**